### DISCUSSION TITLE
Cannot convert to tensorrt with torch.nn.MultiheadAttention

### DISCUSSION THREAD

**Discussion by donghaoye** [open-mmlab/mmclassification#1792] [General]
onnx to tensorrt with "self.attn = torch.nn.MultiheadAttention(dim, 8 )" by using mmdeploy will log the below infos:

[TensorRT] INFO: builtin_op_importers.cpp:3659: Searching for plugin: ScatterND, plugin_version: 1, plugin_namespace:
[TensorRT] INFO: builtin_op_importers.cpp:3676: Successfully created plugin: ScatterND
[TensorRT] WARNING: Output type must be INT32 for shape outputs
[TensorRT] WARNING: Output type must be INT32 for shape outputs
[TensorRT] WARNING: Output type must be INT32 for shape outputs
[TensorRT] WARNING: Output type must be INT32 for shape outputs
[TensorRT] WARNING: Output type must be INT32 for shape outputs
[TensorRT] WARNING: Output type must be INT32 for shape outputs
[TensorRT] WARNING: Output type must be INT32 for shape outputs
[TensorRT] WARNING: Output type must be INT32 for shape outputs
[TensorRT] ERROR: Gather_1639: index to gather must be non-negative
[TensorRT] ERROR: Builder failed while analyzing shapes.
2023-09-12 17:29:36,654 - mmdeploy - ERROR - Failed to create TensorRT engine
2023-09-12 17:29:36,654 - mmdeploy - ERROR - onnx2tensorrt failed.
